app/src/main/java/org/simple/clinic/appupdate/criticalupdatedialog/CriticalAppUpdateUiRenderer.kt

```diff
@@ -17,6 +17,9 @@ class CriticalAppUpdateUiRenderer(
 
     if (model.isCriticalUpdateNudgePriority && model.hasAppStaleness)
       ui.renderCriticalAppUpdateReason(appStalenessInMonths(model.appStaleness!!))
+
+    if (model.isCriticalSecurityUpdateNudgePriority)
+      ui.renderCriticalSecurityAppUpdateReason()
```

Comment:
We are gonna have either a critical update or critical security update priority, but not both right? Should we use an if-else-if / when block here? Because right now both if conditions will run.

Reply:
yeah I thought about this as well. Although, app Update priority value can only be either `CRITICAL_SECURITY` or `CRITICAL` at one point. It can never be both.

Reply:
Yep, so let's just go with a `when` block to simplify it?

Reply:
The if conditionals make more sense here because if this is true we show this or else we do nothing. In a when block we will have to add an empty else case as it wont be exhaustive.

Reply:
Yeah, we can use an `if` block as well. But instead of two different `if` blocks, we can use `if-else-if` to handle the second if condition. Since it's not possible to have 2 different update priorities, we can just have a single conditional block to handle the update priority state. WDYT?